### PR TITLE
First version of models.rst of roman_datamodels docs (RCAL-262)

### PR DESCRIPTION
**Pull request by dmggh**
First version of models.rst - 1 of the 6 rst files for the roman_datamodels doc (epic RCAL-248).

### CODE REVIEW

**Comment by ddavis-stsci** [comment]
Looks like a good start.

File: docs/models.rst [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+----------------------------------
+
+To create a new `WfiImageModel`, just call its constructor. Then populate
+arrays of the new model:
+ 
+    import roman_datamodels.stnode as rds
+    wfi_image = rds.WfiImage()    
+    wfi_image['data'] = np.ones((4096, 4096), dtype=np.float32)
+    wfi_image['dq'] = np.zeros((4096, 4096), dtype=np.uint32)
+    ...
```
Comment (nden) [inline docs/models.rst:45]: This example does not create a data model.

File: docs/models.rst [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  │ └─version (str): 2.5.2a1.dev12+g12aa460
+  ├─history (dict)
+  │ └─extensions (list) ...
+  ├─data (ndarray): shape=(2048, 2048), dtype=float32
+  ├─dq (ndarray): shape=(2048, 2048), dtype=uint32
+  ├─err (ndarray): shape=(2048, 2048), dtype=float32
+  ├─meta (dict)
+  │ ├─aperture (dict) ...
+  │ ├─bunit_data (str): DN/s
+  │ ├─bunit_err (str): DN/s
```
Comment (nden) [inline docs/models.rst:132]: `bunit`?

File: docs/models.rst [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+  │ ├─cal_step (dict) ...
+  │ ├─calibration_software_revision (str): 3bfd782b
+  │ ├─calibration_software_version (str): 0.14.3a1.dev133+g3bfd782b.d20200216
+  │ ├─coordinates (dict) ...
+  │ └─28 not shown
+  ├─var_poisson (ndarray): shape=(2048, 2048), dtype=float32
+  ├─var_rnoise (ndarray): shape=(2048, 2048), dtype=float32
+  └─extra_fits (dict) ...
```
Comment (nden) [inline docs/models.rst:140]: `extrac_fits`?

File: docs/models.rst [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+--------------
+
+Instead of::
+
+    astropy.io.fits.open("myfile.fits")
+
+use::
+
+    from jwst.datamodels import ImageModel
+    with ImageModel("myfile.fits") as model:
```
Comment (nden) [inline docs/models.rst:181]: Why jwst?

**Review by nden — COMMENTED** [review]
Let's talk before you continue with this.

**Comment by dmggh** [comment]
This draft PR has some TBD sections, so still has some JWST artifacts.

**Pull request closed without merging**